integ-test/src/test/java/org/opensearch/sql/sql/SystemFunctionIT.java

```diff
@@ -54,7 +54,7 @@ public void typeof_opensearch_types() {
         //+ ", typeof(nested_value)"
         + " from %s;", TEST_INDEX_DATATYPE_NONNUMERIC));
     verifyDataRows(response,
-        rows("OPENSEARCH_TEXT", "TIMESTAMP", "BOOLEAN", "STRUCT", "STRING",
-                "OPENSEARCH_IP", "OPENSEARCH_BINARY", "OPENSEARCH_GEO_POINT"));
+        rows("text", "TIMESTAMP", "BOOLEAN", "STRUCT", "STRING",
```

Comment:
why are some of these capitalized and some not...?  or does it not matter?

Reply:
This comes from https://github.com/Bit-Quill/opensearch-project-sql/blob/90bae1f0d433aab8982717b9289d1310cfba44ff/opensearch/src/main/java/org/opensearch/sql/opensearch/data/type/OpenSearchDataType.java#L159-L163
This could be changed to have all uppercase.

Reply:
Fixed in b1ffaa3.